storage_controller/src/service.rs

```diff
@@ -4451,7 +4455,7 @@ impl Service {
             waiter_count
         );
 
-        Ok(waiter_count)
+        Ok(std::cmp::max(waiter_count, reconciles_spawned))
```

Comment:
nit: can you update the doc comment too?

Reply:
d4d77d24d